pack/side/blbe_encounter.json

```diff
@@ -1,4 +1,422 @@
 [
+  {
+    "back_flavor": "Very little of Blackwater has been left untouched by the anomaly.",
+    "back_illustrator": "Emilio Rodriguez",
+    "back_name": "Quarantine Zone",
+    "clues": 2,
+    "code": "89001",
+    "double_sided": true,
+    "encounter_code": "blbe",
```

Comment:
This encounter code does not exist, so the set cannot be imported into the database. There is an existing set `blob_that_ate_everything_else` that we can use here.